front-end/src/app/reports/f3x/create-workflow/create-f3x-step1.component.ts

```diff
@@ -74,11 +76,21 @@ export class CreateF3XStep1Component extends FormComponent implements OnInit {
     'state_of_election',
     'form_type',
   ];
-
   readonly form: FormGroup = this.fb.group(SchemaUtils.getFormGroupFieldsNoBlur(this.formProperties, f3xSchema), {
     updateOn: 'blur',
   });
+  readonly reportTypeCategories = [F3xReportTypeCategories.ELECTION_YEAR, F3xReportTypeCategories.NON_ELECTION_YEAR];
+  readonly defaultReportTypeCategory =
+    (this.year % 2 === 0) !== (new Date().getMonth() === 0)
```

Comment:
Can we rewrite this conditional to be more verbose/explicit? It was tough for me to understand what it is trying to do without referencing the requirements.  I'm thinking a slight tweak would be more clear for me:

```
const currMonth = new Date().getMonth();
if (this.year % 2 === 0) {
  defaultReportTypeCategory = currMonth === 0 ?  F3xReportTypeCategories.NON_ELECTION_YEAR : F3xReportTypeCategories.ELECTION_YEAR;
} else {
  defaultReportTypeCategory = currMonth === 0 ?  F3xReportTypeCategories.ELECTION_YEAR: F3xReportTypeCategories.NON_ELECTION_YEAR ;
}
```